src/test/resources/com/atomist/rug/test/gherkin/handler/event/PassingFeature1StepsAgainstGeneratedWithArrays.ts

```diff
@@ -0,0 +1,17 @@
+import {Given,When,Then, HandlerScenarioWorld} from "@atomist/rug/test/handler/Core"
+
+import {Push} from "@atomist/rug/cortex/stub/Push"
+import {Commit} from "@atomist/rug/cortex/stub/Commit"
+
+ 
+Given("a sleepy country", f => {
+})
+When("a visionary leader enters", world => {
+   world.registerHandler("ReturnsEmptyPlanEventHandlerGenWithArrays")
+   let p = new Push().addContains(new Commit())
```

Comment:
`addCommit` can't work consistently as there might be multiple properties relating to the `Commit` type.

I don't like exposing the array and adding to it directly as it starts off being `undefined` rather than empty. Our `add` methods handle that correctly.

I think we'll learn more over time, but the `add` methods seem the best option we've concerned so far.